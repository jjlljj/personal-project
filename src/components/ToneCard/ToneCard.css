.tone-card {
  width: 260px;
  min-height: 200px;
  margin: 20px;
  border: 1px solid #f2f2f2;
}

.tone-title-wrap {
  width: 100%;
  height: 50px;
  position: relative;
  display: flex;
  align-items: center;
  justify-content: center;
}

.tone-title-wrap .overlay {
  position: absolute;
  top: 0;
  left: 0;
  height: 50px;
  width: 100%;
  background: white;
  opacity: .25;
}

.tone-card p {
  margin: 20px;
}

.tone-title-wrap h3 {
  z-index: 1;
}

.background-Sadness {
  background-color: purple;
}

.background-Analytical {
  background-color: green;
}

.background-Tentative {
  background-color: #4CCEE8;
}

.background-Confident {
  background-color: magenta;
}

.background-Angry {
  background-color: red;
}
.background-Fear {
  background-color: orange;
}

.background-Positive {
  background-color: blue;
}

.background-Neutral {
  background-color: #6d6c6c;
}
