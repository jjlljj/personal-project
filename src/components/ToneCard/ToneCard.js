import React from 'react'
import './ToneCard.css'

export const ToneCard = ({ tone_name, description, score = "" }) => {

  const renderedScore = score && (
      <h4>{score}</h4>
    ) || ""

  return (
    <div className="tone-card">
      <div className={ "tone-title-wrap background-" + tone_name }>

        <h3>{tone_name}</h3>
        <div className="overlay"></div> 
      </div>
      { renderedScore }
      <p>{description}</p>
    </div>
  )
}

export default ToneCard