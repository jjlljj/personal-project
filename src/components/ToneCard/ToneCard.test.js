/* eslint-disable */
import React from 'react'
import { shallow } from 'enzyme'
import { ToneCard } from './ToneCard'

describe('ToneCard', () => {
  let renderedComponent

  beforeEach(() => {
    renderedComponent = shallow(<ToneCard />)
  })

  it('should match snapshot', () => {
    expect(renderedComponent).toMatchSnapshot()
  })
})
