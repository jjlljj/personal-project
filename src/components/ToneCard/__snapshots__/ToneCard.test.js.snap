// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToneCard should match snapshot 1`] = `undefined`;
